src/screens/welcome/Welcome.module.css

```diff
@@ -0,0 +1,48 @@
+.bg{
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  height: 100vh;
+  padding-top: 11vh;
+  padding-bottom: 8vh;
+  background: fit-content;
+}
+
+.welcome_title {
+  margin-left: 10vw;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 36px;
+  line-height: 40px;
+  text-align: left;
+  color: #151515;
+  background-color: rgba(227, 184, 31, 0.5);
+  padding: 1px 16vw 4px 15px;
+}
+
+.bottom_section{
+  display: flex;
+  flex-direction: column;
+  
+  .welcome_hint{
+    background-color: rgba(227, 184, 31, 0.5);
+    font-size: 20px;
+    margin-right: 20vw;
+    text-align: left;
+    padding-left: 25vw;
+    padding-top: 10px;
```

Comment:
Use `rem` here too?

Reply:
This is the padding of the bottom yellow box surrounding texts, I think it should be kept fixed but if you mean the font size of those texts I've changed it.